### DISCUSSION TITLE
CC3200 port on LAUNCHXL ?

### DISCUSSION THREAD

**Discussion by niallp** [micropython/micropython#10202] [Other] [answered]
I know it isn't a popular port but is there anyone else using it (original WiPy as well) ? I recently tried an update to the latest master (v1.19.1-740-gbf49a087b) and while it compiles cleanly it does not appear to run (can't get REPL on the serial port).

Last version I used was a while ago (v1.9.3-7-gad5a6f5) but jumping back to that breaks the compile. While I had all this working a couple years ago, systems and compilers have been upgraded along the way.

**Comment by robert-hh** [comment, marked as answer]
I could build & flash the actual repo state with an WiPy board. I had to reset the board after upload, to get a REPL at /dev/ttyUSB0.

**Reply by niallp** [reply]
Thanks for checking, much appreciated ! 
Which version of arm-none-eabi-gcc were you using ?
 I'm on Linux Mint 21 so installed it from the repos: _arm-none-eabi-gcc (15:10.3-2021.07-4) 10.3.1 20210621 (release)_

**Reply by robert-hh** [reply]
This is the version I used with my Debian 9 system:
`gcc (Debian 10.2.1-6) 10.2.1 20210110`

But using `gcc-arm-none-eabi-10.3-2021.07` works as well.

**Reply by niallp** [reply]
Sounds like it should work though still nothing on the console. I've managed to transfer the old `mcuimg.bin` and `factimg.bin` back onto the LAUNCHXL and confirmed it works again so not the new servicepack.

Looks like I may be stuck with the old version till I can figure out how to see what is wrong with the latest version on boot. Unfortunately don't have a serial boot console to watch, possibly someway to use builtin JTAG ? 

@robert-hh any experience with the Launchpad itself (instead of WiPy) ? Prebuilt images for WiPy are available but not Launchpad.

**Reply by niallp** [reply]
Managed to get it updated via the OTA method (FTP to /flash/sys) ! 

The Launchpad now running MicroPython **v1.19.1-740-gbf49a087b-dirty** on 2022-12-12; LaunchPad with CC3200

Maybe some issue with the bootloader (doesn't get updated this way) but works for now :)

**Reply by robert-hh** [reply]
I used always the OTA method. Just did not mention it.
